index.js

```diff
@@ -106,51 +141,49 @@ proto.shot = function( opts, next ){
   }
 
   return Promise.try(function(){
-    return browserifyPhantomSrc();
+    return writeExtensionsList();
   }).then(function(){
-    return snap.phantom.createPage();
-  }).then(function( phantomPage ){
-    page = phantomPage;
+    return browserifyBrowserSrc();
   }).then(function(){
-    return page.property('viewportSize', { width: opts.width, height: opts.height });
+    return snap.browser.newPage();
+  }).then(function( puppeteerPage ){
+    page = puppeteerPage;
```

Comment:
what do you think of caching page so that we don't have to create a newPage every time someone wants a snapshot?

https://github.com/GoogleChrome/puppeteer/issues/1569
https://github.com/GoogleChrome/puppeteer/issues/486

Reply:
I wanted to maintain separation for two reasons:

(1) You can generate multiple images at once.

(2) It helps to avoid potential bugs.  For example, if there's a bug in Cytoscape when running for a long time with multiple stylesheets etc., then one page per output avoids it.  We're also setting a lot of globals in the page, which could be messy re. re-use.

The most important one is (1), whereas (2) is relatively unlikely.